resources/knative-eventing-kafka/templates/serviceentry.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: {{ $.Release.Name }}
```

Comment:
IMO `knative-kafka.fullname` is the safest thing to use because of the added logic used to derive the value. But I see we haven't used it anywhere.